### DISCUSSION TITLE
Can you size a section?

### DISCUSSION THREAD

**Discussion by gvlx** [Budibase/budibase#3303] [Help]
A section element splits a container in two or more slices (columns) but I can't understand how to fit each one of the to a certain size.

**Comment by mjashanks** [comment]
It's not possible with sections, they are meant to be a quick and easy way to lay things out, but lack that flexibility as a result. They are also responsive (mobile & tablet friendly), which doesn't do well with fixed sizes.

However, you can use a container the achieve the same thing....

1. Add a new container
2. Set it's `Direction = Row` - this means that it's children will get added **across** the screen
3. Add 2 containers inside
4. For the first container, set `Size = Shrink`
5. For the second container, set `Size = Grow`
6. You can now set the width of the child containers to whatever you like. As mentioned, this won't size well on mobile - although you can control this with `Conditions`, if you need.

Your designer should look like this

![image](https://user-images.githubusercontent.com/3524181/141084967-8dee9bad-89e7-4d2b-978f-0cdcdfd6f8db.png)
